.github/workflows/pull_request.yml

```diff
@@ -8,6 +8,10 @@ on:
     branches:
       - main
 
+env:
+  ARC_TABLES_PORT: 8000
+  ARC_DB_EXTERNAL: true
```

Comment:
I think that you only want to set these environment variables for the Playwright tests job, right? Not all the unit tests.

Reply:
Do you want me to review this now, or are you still working on it?

Reply:
I'm still working on it, But i got it mostly there. I think there might need to be some checks for whether this is a ci thing so that it isn't actually producing kafka messages. Progress is progress